app/web/utils/SearchFilters.ts

```diff
@@ -1,61 +1,15 @@
-import { useRouter } from "next/router";
 import { ParsedUrlQuery } from "querystring";
```

Comment:
this file (and test file) should be uncapitalized

Reply:
gotcha, cool. I wasn't sure since the main export was `SearchFilters` type, but there are other things too, so it makes sense lower case